src/backend/language.cpp

```diff
@@ -79,238 +80,1049 @@ LanguageMap::LanguageMap() {
+    addLanguage({"toj"}, QT_TRANSLATE_NOOP("QObject", "Tojolabal"));
+    addLanguage({"too"}, QT_TRANSLATE_NOOP("QObject", "Xicotepec De Juárez Totonac"));
+    addLanguage({"top"}, QT_TRANSLATE_NOOP("QObject", "Papantla Totonac"));
+    addLanguage({"tos"}, QT_TRANSLATE_NOOP("QObject", "Highland Totonac"));
+    addLanguage({"tpa"}, QT_TRANSLATE_NOOP("QObject", "Taupota"));
+    addLanguage({"tpi"}, QT_TRANSLATE_NOOP("QObject", "Tok Pisin"));
+    addLanguage({"tpt"}, QT_TRANSLATE_NOOP("QObject", "Tlachichilco Tepehua"));
+    addLanguage({"tpz"}, QT_TRANSLATE_NOOP("QObject", "Tinputz"));
     addLanguage({"tr"}, QT_TRANSLATE_NOOP("QObject", "Turkish"));
+    addLanguage({"trc"}, QT_TRANSLATE_NOOP("QObject", "Copala Triqui"));
     //  addLanguage({"ts"}, QT_TRANSLATE_NOOP("QObject", "Tsonga"));
+    addLanguage({"tsg"}, QT_TRANSLATE_NOOP("QObject", "Tausug"));
+    addLanguage({"tsw"}, QT_TRANSLATE_NOOP("QObject", "Tsishingini"));
     //  addLanguage({"tt"}, QT_TRANSLATE_NOOP("QObject", "Tatar"));
+    addLanguage({"tt-Cyrl"}, QT_TRANSLATE_NOOP("QObject", "Tatar (Cyrillic script)"));
     addLanguage({"ttc"}, QT_TRANSLATE_NOOP("QObject", "Tektiteko"));
-    //  addLanguage({"tw"}, QT_TRANSLATE_NOOP("QObject", "Twi"));
+    addLanguage({"tte"}, QT_TRANSLATE_NOOP("QObject", "Bwanabwana"));
+    addLanguage({"tuc"}, QT_TRANSLATE_NOOP("QObject", "Mutu"));
+    addLanguage({"tue"}, QT_TRANSLATE_NOOP("QObject", "Tuyuca"));
+    addLanguage({"tuf"}, QT_TRANSLATE_NOOP("QObject", "Central Tunebo"));
+    addLanguage({"tuo"}, QT_TRANSLATE_NOOP("QObject", "Tucano"));
+    addLanguage({"tvk"}, QT_TRANSLATE_NOOP("QObject", "Southeast Ambrym"));
+    // addLanguage({"tw"}, QT_TRANSLATE_NOOP("QObject", "Twi"));
+    addLanguage({"txq"}, QT_TRANSLATE_NOOP("QObject", "Tii"));
+    addLanguage({"txu"}, QT_TRANSLATE_NOOP("QObject", "Kayapó"));
     addLanguage({"ty"}, QT_TRANSLATE_NOOP("QObject", "Tahitian"));
+    addLanguage({"tyv"}, QT_TRANSLATE_NOOP("QObject", "Tuvinian"));
+    addLanguage({"tzj"}, QT_TRANSLATE_NOOP("QObject", "Tz'utujil"));
+    addLanguage({"tyv-Cyrl"}, QT_TRANSLATE_NOOP("QObject", "Tuvinian (Cyrillic script)"));
+    addLanguage({"tzo"}, QT_TRANSLATE_NOOP("QObject", "Tzotzil"));
     addLanguage({"tzz"}, QT_TRANSLATE_NOOP("QObject", "Tzotzil, Zinacant\u00e1n"));
-    //  addLanguage({"ug"}, QT_TRANSLATE_NOOP("QObject", "Uighur"));
+
     addLanguage({"uk"}, QT_TRANSLATE_NOOP("QObject", "Ukrainian"));
-    //  addLanguage({"ur"}, QT_TRANSLATE_NOOP("QObject", "Urdu"));
     addLanguage({"ury"}, QT_TRANSLATE_NOOP("QObject", "Orya"));
     addLanguage({"usp"}, QT_TRANSLATE_NOOP("QObject", "Uspanteco"));
-    //  addLanguage({"uz"}, QT_TRANSLATE_NOOP("QObject", "Uzbek"));
+    addLanguage({"ubr"}, QT_TRANSLATE_NOOP("QObject", "Ubir"));
+    addLanguage({"ubu"}, QT_TRANSLATE_NOOP("QObject", "Umbu-Ungu"));
+    addLanguage({"udu"}, QT_TRANSLATE_NOOP("QObject", "Uduk"));
+    addLanguage({"ug"}, QT_TRANSLATE_NOOP("QObject", "Uyghurche‎"));
```

Comment:
This is Uighur (Uyghur), and needs an "uig" abbreviation according to https://iso639-3.sil.org/code/uig